src_py/sysfont.py

```diff
@@ -204,35 +195,39 @@ def initsysfonts_unix(path="fc-list"):
     fonts = {}
 
     try:
-        # pylint: disable=consider-using-with
```

Comment:
I wonder if the changes were checked with pylint?

Reply:
Thanks for the review! 🥳 

Here pylint does not complain because [`subprocess.run`](https://docs.python.org/3/library/subprocess.html#subprocess.run) does not export a context-manager API like the lower level `Popen` constructor that the old code used.